ParquetSharp.Dataset.Test/ParquetSharp.Dataset.Test.csproj

```diff
@@ -10,6 +10,7 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.3.2"/>
+    <PackageReference Include="Newtonsoft.Json" Version="13.0.3" />
```

Comment:
I think it is not needed, or is it?

Reply:
It might work in CI without this, but when I tried to install a new Arrow version locally in Rider it complained about a security vulnerability in an older Newtonsoft.Json version (9 point something I think).

I'll revert this and see what happens.